Source/WebKit/UIProcess/_WKWebViewPrintFormatter.mm

```diff
@@ -96,17 +128,19 @@ - (CGRect)rectForPageAtIndex:(NSInteger)pageIndex
 
 - (void)drawInRect:(CGRect)rect forPageAtIndex:(NSInteger)pageIndex
 {
-    if (!_printedDocument) {
-        _printedDocument = self._webView._printProvider._wk_printedDocument;
-        if (!_printedDocument)
+    CGPDFDocumentRef printedDocument = [self _printedDocument];
```

Comment:
Would feel safer to retain the printedDocument

Reply:
Done.